dimagi/pages/templates/sections/toolkits/summary.html

```diff
@@ -53,7 +53,17 @@ <h4>
                 Download {{ toolkit.title }}
               {% endif %}
             </h4>
-            <div id="hubspot-download-form-summary" class="hubspot-download"></div>
+            <div class="gap-top-small">
+              <p class="button download-btn-toolkit"
+                   href="#toolkit-summary-modal">
+                  <span>
+                    {% blocktrans %}
+                      Download
+                    {% endblocktrans %}
+                  </span>
+                </p>
+            </div>
+            <!-- <div id="toolkit-summary-modal" class="hubspot-download"></div> -->
```

Comment:
@hemanthReddy06 can you clean this comment up please? :)

Reply:
@hemanthReddy06 re-ping

Reply:
@biyeun Updated.